core/templates/pages/exploration-player-page/layout-directives/progress-nav.directive.html

```diff
@@ -48,7 +48,6 @@
 
   progress-nav .progress-nav-bar {
     background-color: white;
-    margin-left: 87px;
```

Comment:
Hm, can you explain why this is required?

Reply:
I needed to expand the card, so this margin was making the bottom part shrink
![image](https://user-images.githubusercontent.com/30312043/78775895-9c34c680-79b4-11ea-97dc-995104bc2c18.png)
![image](https://user-images.githubusercontent.com/30312043/78775948-ae166980-79b4-11ea-9a06-f0a1092aaf99.png)

Reply:
I see. sg, thanks!